### DISCUSSION TITLE
Market Provider

### DISCUSSION THREAD

**Discussion by tjitra** [Capnode/Algoloop#24] [Q&A] [answered]
Im not familiar with wpf im still use VB, sory for my bad.
Were i can find setting of market provider, like, binance, avanza, bybit etc.
Btw i want to use yahoo finance data. I have search from the code but cant find were is it.
Thankyou for help

**Comment by Capnode** [comment, marked as answer]
Most market providers has not been tested in Algoloop and in some cases they just refer to the code provided by QuantConnect. No guarantee that they will work. The only providers that has been verified are Metastock and Borsdata.
